src/egl/drivers/dri2/egl_dri2.h

```diff
@@ -58,6 +58,7 @@
 
 #include <system/window.h>
 #include "platform_android_gralloc_drm.h"
+#include <hardware/gralloc1.h>
```

Comment:
For upstreaming we need to protect these gralloc1 things with 'ANDROID_API_LEVEL'  checks. I can take care of this later unless you feel like doing it. This is because Mesa supports Android versions that don't have the header in place.